src/app/equipment/equipment.cpp

```diff
@@ -0,0 +1,12 @@
+#include "equipment.hpp"
+
+Equipment::Equipment(const std::string &name, const double price, const int amount, const std::string &material,
+                     const std::string &description)
+    : Product(name, price, amount), material_(material), description_(description)
+{
+}
```

Comment:
Please use initializer list in constructor.

Reply:
Done